BitwardenShared/Core/Auth/Services/API/Organization/Fixtures/SingleSignOnDetailsNoVerifiedDate.json

```
@@ -1,5 +1,4 @@
 {
   "OrganizationIdentifier": "TeamLivefront",
-  "SsoAvailable": true
+  "SsoAvailable": true,
```

⛏️ This shouldn't have a trailing comma for JSON:
```suggestion
  "SsoAvailable": true
```